modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must specify the projects associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same project as the `ServiceMeshControlPlane` resource and name it `default`.
```

Comment:
Strange sentence - starts off with "must specify", but finished with "and name it", which would make more sense if the sentence started with "must create a SMMR resource named default in order to specify the projects associated..."

Reply:
I changed this to read:

"You must create a `ServiceMeshMemberRoll` resource named `default` associated with the {ProductShortName} in the same project as the `ServiceMeshControlPlane`."